go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	k8s.io/api v0.18.6
 	k8s.io/apimachinery v0.18.6
 	k8s.io/client-go v12.0.0+incompatible
+	k8s.io/kubernetes v1.17.4
```

Comment:
Is this all that was needed to be able to register the CassandraDatacenter scheme in the integration tests?